App.js

```diff
@@ -77,6 +83,42 @@ class App extends Component {
     LayoutAnimation.spring();
   }
 
+  getCurrentRouteName(navigationState) {
+    if (!navigationState) {
+      return null;
+    }
+    const route = navigationState.routes[navigationState.index];
+
+    if (route.routes) {
+      return this.getCurrentRouteName(route);
+    }
+
+    return route.routeName;
+  }
+
+  statusBarHandler(prev, next) {
+    const routeName = this.getCurrentRouteName(next);
+    const lightContentScreens = [
+      'MyProfile',
+      'Profile',
+      'Organization',
+      'Repository',
+      'Login',
+    ];
+
+    if (lightContentScreens.includes(routeName)) {
+      StatusBar.setTranslucent(true);
+      StatusBar.setBackgroundColor(
+        routeName === 'Login' ? colors.transparent : colors.black
+      );
+      StatusBar.setBarStyle('light-content');
+    } else {
+      StatusBar.setTranslucent(false);
+      StatusBar.setBackgroundColor(colors.grey);
+      StatusBar.setBarStyle('dark-content');
+    }
```

Comment:
@jouderianjr Can we config these informations in somewhere? So that we will not forget to update them once new routers were added.